### ISSUE TITLE
Simple search always returns error code 500

### ISSUE THREAD

**Issue by TeamArty** [Scrip7/fmovies-crawler#4]
When i try `/search/simple` it always returns error code 500

I found out that in console it gets the code 403 meaning something is Forbidden

Looks like FMovies implemented a new param called vrf and if its not right, the search will return error code 403
https://fmovies.to/search?keyword=blacklist&vrf=irRPMrMX2%2F4K%2FhpZ04 works
and https://fmovies.to/search?keyword=blacklist doesn't work

Is there any way to fix this?

**Comment by Scrip7** [comment]
Hey @TeamArty, Thanks for explaining in detail!

Yeah, it seems like they implemented the `vrf` query parameter to secure their resources.

The simple search endpoint (`/search/simple`) sends an [AJAX request](https://github.com/Scrip7/fmovies-crawler/blob/be2b5abbafd16464aaa2f2fc4ef1fbd62d203c40/src/request/request.service.ts#L38) to scrape the data.
To fix this issue, it must use the Selenium driver instead as the way the `/video/details` endpoint does.

Unfourtualy, as I said [here](https://github.com/Scrip7/fmovies-crawler/issues/3#issuecomment-907367195) in the previous issue,
> I don't have enough time to maintain this project anymore.

Maybe someday in the future, I'll rewrite everything using [puppeteer](https://github.com/puppeteer/puppeteer) to fix all the known issues.
Till then, PRs are welcome tho 😄

**Comment by TeamArty** [comment]
Hey, where can I find how `/video/details` endpoint does it? What file and function?

**Comment by TeamArty** [comment]
How would I get the `vrf`?

Is selenium able to use a html form? Or something else

**Comment by Scrip7** [comment]
Yes, Whatever you do with a regular browser, can also be done with Selenium!
First, you navigate to the `https://fmovies.to/` URL, fill up the search input on top and extract the result using JavaScript.
In simpler words, it automates the browser.
